packages/fxa-auth-server/test/mail_helper.js

```diff
@@ -40,7 +44,7 @@ async function printMatchingKeys(startUp = false) {
       }
     }
   } catch (error) {
-    console.error('Failed to retrieve keys:', error);
+    // Ignore errors
```

Comment:
Do we really want to suppress this?

Reply:
In my testing it could fail to retrieve keys if not yet connected to redis. It would get called again on after the setTimeout. Figured this was fine since its only test code.